src/client/src/services/connection.ts

```diff
@@ -46,8 +46,12 @@ const connectionExists = (status: ConnectionStatus, dispose: Disposable) => {
 
 // Connect to Hydra gateway and store the disposable
 export const initConnection = async () => {
+  const url = import.meta.env.VITE_LOCAL_HYDRA_URL
```

Comment:
should not need to change the app code ... see https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/client/vite.config.ts#L338